modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -165,6 +165,84 @@ definition {
 		AppBuilderAdmin.assertNewButtonIsVisible(buttonValue = "New Form View");
 	}
 
+	@description = "This is a use case for LPS-95964"
+	@priority = "5"
+	test AddRulesViaFormView {
+		property test.name.skip.portal.instance = "AppBuilder#AddRulesViaFormView";
+		property portal.acceptance = "true";
+
+		var myFormViewName = "My Registration form";
+		var myTableViewName = "My display table";
+		var myAppName = "My Registration app";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Text");
```

Comment:
Hi @calvinctvo , could you please change the default label of Text? It is important because this label should be shown at the Rules modal (when creating a rule) and in the Rules summary after the creation, the place where I asked you to also validate the rule summary.